docs/observers.rst

```diff
@@ -591,6 +592,40 @@ Schema
 .. image:: images/sql_schema.png
 
 
+.. _s3_observer:
+
+S3 Observer
+============
+The S3Observer stores run information in a designated prefix location within a S3 bucket, either by
+using an existing bucket, or creating a new one. Using the S3Observer requires that boto3 be
+installed, and also that an AWS config file is created with a user's Access Key and Secret Key.
+An easy way to do this is by installing AWS command line tools (``pip install awscli``) and
+running ``aws configure``.
+
+Adding a S3Observer
+--------------------
+
+To create an S3Observer in Python:
+
+.. code-block:: python
+
+    from sacred.observers import S3Observer
+    ex.observers.append(S3Observer.create(bucket='my-awesome-bucket',
```

Comment:
Please then also update the docs to use `__init__` instead of `create`.

Reply:
A slightly tangential question: I notice as I'm editing the docs that they don't seem to presently be up to date for the existing observers (i.e. the current master docs say "Sacred ships with four observers" and then has a list that doesn't include the Slack, Telegram, or Neptune observers. I feel slightly weird about continuing the error to say "Sacred ships with five observers", but also don't feel conversant enough about how the Slack/Telegram/Neptune observers work to add the summary documentation for them, so by default I'll just up the number to five and add the S3Observer to the list, but just flagging this as something I wanted to make sure was intentional on the part of the maintainers.